src/main/java/nextstep/ladder/model/Point.java

```diff
@@ -0,0 +1,20 @@
+package nextstep.ladder.model;
+
+import java.util.Random;
+
+public class Point {
+    private final Random random = new Random();
```

Comment:
```suggestion
    private static final Random random = new Random();
```
`Random`을 계속해서 생성할 필요가 있을까요?